[docs-agent] Deprecate Account Kit (v4) in wallets nav (#1530)
alchemyplatform/docs: content/wallets/pages/transactions/solana/sponsor-gas-solana.mdx

Fees and rent are a significant barrier to entry for new users of your app. Sponsor them to let users transact without holding SOL.

> * Works with any Solana wallet. [Check out Solana Wallet APIs](/docs/wallets/solana/signers)

Reviewer: ```suggestion
* Works with any Solana wallet. [Check out Solana Wallet APIs](/docs/wallets/react/solana-wallets/get-started)
```

## Prerequisites

* API key from your [dashboard](https://dashboard.alchemy.com/apps)

> * Wallet APIs for Solana [set up](/docs/wallets/solana/signers) in your project if you want to enable sign up/login for creation of wallets

Reviewer: ```suggestion
* Wallet APIs for Solana [set up](/docs/wallets/react/solana-wallets/get-started) in your project if you want to enable sign up/login for creation of wallets
```

> * Wallet APIs for Solana [set up](/docs/wallets/solana/signers) in your project if you want to enable sign up/login for creation of wallets

Reviewer: ```suggestion
```
